.travis.yml

```diff
@@ -59,3 +59,15 @@ matrix:
       - unzip protoc-3.5.1-linux-x86_64.zip -d protoc3.5
       - PATH=$(pwd)/protoc3.5/bin:${PATH}
     script: check/pytest2
+
+  - os: windows
+    language: sh
+    python: "3.5.2"
+    before_install:
+    - choco install python3
```

Comment:
You've confirmed this step is actually needed? I would have expected that asking for python 3.5.2 above to ensure it was installed.

Reply:
Just confirmed that choco is necessary.  The `python: "3.5.2"` displays what version is being run on the dashboard.